src/main/java/ee/ut/App.java

```diff
@@ -60,8 +61,17 @@ public static void startTimeline(boolean isNew) throws IOException {
         if (isNew) {
             data = new Data();
         } else {
-            //todo implement loading data from js
-            throw new IllegalArgumentException("Editing and existing timeline not yet implemented");
+
+            FileChooser fileChooser = new FileChooser();
```

Comment:
Please consider making it so the FileChooser's initial directory is the "result" directory

Reply:
Added changes